src/Files.App/Views/SettingsPages/Appearance.xaml

```diff
@@ -63,18 +64,21 @@
 						</GridView.ItemTemplate>
 					</GridView>
 				</local:SettingsBlockControl.ExpandableContent>
-				<Button
-					x:Name="ThemesLearnMoreButton"
-					Height="32"
-					Padding="8,4,8,4"
-					VerticalAlignment="Bottom"
-					AutomationProperties.Name="{helpers:ResourceString Name=SettingsThemesLearnMoreButton/AutomationProperties/Name}"
-					Background="Transparent"
-					BorderBrush="Transparent"
-					Click="ThemesLearnMoreButton_Click"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=SettingsThemesLearnMoreButton/ToolTipService/ToolTip}">
-					<FontIcon FontSize="14" Glyph="&#xE946;" />
-				</Button>
+				<StackPanel Orientation="Horizontal">
+					<ProgressRing x:Name="AppThemeLoading" Height="32" Width="32" Padding="8" VerticalAlignment="Bottom" Visibility="Collapsed" IsIndeterminate="True" />
```

Comment:
Should this have an automation name for screen readers?

Reply:
Is there an existing localized string for such loading indicator?

Reply:
Not yet, let's add one for "Loading themes".

Reply:
What's the workflow of adding a new string now?

Reply:
I've added one `SettingsThemesLoadingProgressRing/AutomationProperties/Name`.

Reply:
If you add a string to the en-us resource file called "LoadingThemes", Crowdin will handle the other resource files after this PR is merged.